src/fides/api/models/privacy_preference.py

```diff
@@ -1,5 +1,11 @@
 # pylint: disable=R0401, C0302, W0143
+"""
+TODO SQLAlchemy models in this file DeprecatedCurrentPrivacyPreference and DeprecatedLastServedNotice
```

Comment:
ok, nice to have this clear warning now!

Reply:
This ticket to cleanup keeps getting deprioritized but hopefully I can knock it out this sprint!  I was reminded as part of this work that columns in these deprecated tables are referencing privacy notice history columns, and I'd love to get this cleaned up asap.